src/widgets/PageInfo/PageInfo.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { FlexRow } from '../FlexRow';
+import { PageInfoColumn } from './PageInfoColumn';
+
+import { SUSSOL_ORANGE, DARK_GREY } from '../../globalStyles';
+
+/**
+ * A component to display info in a generic way at the top of a page.
+ * @prop  {bool}   isEditingDisabled  Indicator whether all editing is disabled
+ * @prop  {string} infoColour         Colour of all info components
+ * @prop  {string} titleColour        Colour of all title components
+ * @prop  {array}  columns   An array containing columns of information to be
+ *                          displayed, with an entry in the array representing
+ *                          a column, which is an array of info objects
+ *                          containing a title and info.
+ *
+ *                          E.g.:
+ *
+ *                          [[{title: 'col1:', info: 'row1'},
+ *                            {title: 'col1:', info: 'row2'}]
+ *                           [{title: 'col2:', info: 'row1',
+ *                             editableType: 'selectable'},
+ *                            {title: 'col2:', info: 'row2',
+ *                             editableType: 'text'}]]
+ *
+ *                          would display:
+ *
+ *                            col1: row1 col2: row1
+ *                            col1: row2 col2: row2
+ */
+export const PageInfo = React.memo(({ columns, isEditingDisabled, titleColour, infoColour }) => (
```

Comment:
So.. uhh.. I was looking through `PageInfo` as I was adding the date type, and I editted it slightly as there were un-needed styles and conditionals etc as well as it having had a bunch of code added since it was first created and it was a bit messy. I.e. a field `shouldHide` in columns which was being filtered on, but it wasn't actually used anywhere and by the time I was finished editing it was easier to just refactor a little!